config/locales/en.yml

```diff
@@ -1915,13 +1914,31 @@ en:
       start: "Start"
       end: "End"
       activity: "Activity"
-      room: "Room"
+      no_activities: "No activities for the selected rooms/events."
+      all_venues: "All Venues"
+      room_or_stage: "Room or Stage"
+      more_details: "Show Round Details"
+      time_zone: "Time Zone"
+      rooms_panel:
+        title: "Rooms"
+        all: "All"
+        none: "None"
+        show_buttons: "Show time zone buttons"
+        use_time_zone: "Use this time zone"
       #context: when displaying the room on the general schedule, we may also display the venue if there are multiple venues
       room_in_venue: "%{room} in %{venue}"
-      venue_information_html: "You are viewing the schedule for the venue %{venue_name}."
-      timezone_message: "The schedule is displayed in the timezone %{timezone}."
-      multiple_venues_available: "This competition has multiple venues, make sure to check out the schedule for the other venues as well!"
-      schedule_for_date: "Schedule for %{day_name} (%{full_date})"
+      venue_information_html:
+        one: "You are viewing the schedule for the venue %{venue_name}, the sole venue for this competition."
+        other: "You are viewing the schedule for the venue %{venue_name}, one of %{count} venues for this competition."
+      venue_information_all: "You are viewing the schedule for all %{venueCount} venues at once."
+      timezone_message: "The time zone for this venue is %{timezone}."
+      timezone_setting: "You are viewing this schedule in the following timezone setting:"
+      multiple_timezones_available: "Note that not all venues are in the same time zone -- please be careful!"
+      timezone_set_local: "Set to local"
+      timezone_set_venue: "Set to venue"
+      timezone_follow_venue: "Follow the active venue tab"
```

Comment:
This wording makes sense once I know the functionality that it's referring to, but when I first saw it I was slightly confused. I thought it was something (maybe notification based? that's what 'follow' usually means online) to do with the specific _current_ active venue tab. I think something along the lines of "Automatically show the active venue's time zone" might be better. It might just be me though.